modules/Router.js

```diff
@@ -19,6 +19,7 @@ class Router extends Component {
     history: object,
     children: routes,
     routes, // alias for children
+    RoutingContext: func.isRequired,
```

Comment:
I suppose it would be different if the DI here was done via module. But then you wouldn't even have this problem in the first place (you'd have others, but that's a different story...)